odpc.client/src/features/publicatie/PublicatieDetails.vue

```diff
@@ -87,9 +117,27 @@ const {
   // Publicatie.uuid is used when new pub and associated docs: docs submit waits for pub submit/publicatie.uuid.
   useDocumenten(computed(() => props.uuid || publicatie.value?.uuid));
 
+const navigate = () => {
+  if (previousRoute.value?.name === "publicaties") {
+    router.push({ name: previousRoute.value.name, query: previousRoute.value?.query });
+  } else {
+    router.push({ name: "publicaties" });
+  }
+};
+
 const submit = async () => {
   if (validateForm(formRef.value).invalid) return;
 
+  if (publicatie.value.status === PublicatieStatus.ingetrokken) {
+    const { isCanceled } = await dialog.reveal();
+
+    if (isCanceled) {
+      publicatie.value.status = PublicatieStatus.gepubliceerd;
```

Comment:
Vanuit de gebruikersinterface is het heel logisch dat je de status hier terugzet op gepubliceerd, want de knop waarmee je de dialog annuleert heeft als tekst _Nee, gepubliceerd laten_. Maar op deze plek in de code is dat niet duidelijk. Wellicht is een comment voldoende om dit te verduidelijken.

Reply:
Comment toegevoegd.